compiler/test/formatter_inputs/includes.gr

```diff
@@ -0,0 +1,32 @@
+module Includes
+
+include "runtime/unsafe/memory"
+include "runtime/unsafe/tags"
+include "list"
+include "option" as Opt
+// TODO(#1627): This comment disappears
+include /* special include */ "array"
+from List use { length, map, forEach as each }
+from Opt use { MutableOpt, ImmutableOpt as Imm, type Opt, type Opt as OptAlias }
```

Comment:
We should probably toss some comments in these too?

Reply:
done